src/test/java/org/opensearch/security/privileges/PrivilegesEvaluatorUnitTest.java

```diff
@@ -8,14 +8,95 @@
 
 package org.opensearch.security.privileges;
 
+import com.google.common.collect.ImmutableList;
 import org.junit.Test;
 
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.opensearch.security.privileges.PrivilegesEvaluator.DNFOF_PATTERNS;
 import static org.opensearch.security.privileges.PrivilegesEvaluator.isClusterPerm;
 
 public class PrivilegesEvaluatorUnitTest {
 
+    private static final List<String> allowedDnfof = ImmutableList.of(
```

Comment:
I just copied a list of all index permissions found on the documentation website - am I understanding DNFOF correctly in that its only an index level permission filter?

Reply:
Confirmed that this is the case